packages/material-ui/src/Tab/Tab.d.ts

```diff
@@ -8,7 +8,7 @@ export type TabTypeMap<P = {}, D extends React.ElementType = 'div'> = ExtendButt
     fullWidth?: boolean;
     icon?: string | React.ReactElement;
     label?: React.ReactNode;
-    onChange?: (event: React.ChangeEvent<{ checked: boolean }>, value: any) => void;
```

Comment:
I don't know why the target should implement `checked`. Probably some wrong c&p while refactoring.